packages/deno/src/executors/emit/emit.impl.ts

```diff
@@ -79,15 +80,18 @@ function normalizeOptions(
   options.bundle ??= true;
   options.assets ??= [];
 
+  options.sourceMap ??= 'inline';
+  if (options.sourceMap === true) {
+    options.sourceMap = 'inline';
+  }
+
   return options;
 }
 
 function createArgs(options: BuildExecutorSchema, context: ExecutorContext) {
   const tmpBundleFile = createTempEmitFile(options, context);
   const args = ['run', '--allow-all'];
 
-  args.push(`--config=${options.denoConfig}`);
```

Comment:
is there any reason why we still shouldn't provide the config file for `deno run` command? presumably it doesn't matter, but just wanted to double check.

Reply:
the config really only applies to the user's code, the deno instance we're running here is not running the user's code but instead an nx script. If we applied the deno config, it would be as if we were applying the user's tsconfig to the code in the nx npm package.